src/services/localStorage.tsx

```diff
@@ -0,0 +1,38 @@
+import {Dispatch} from "react";
+import {TeamType} from "./getTeamTokenMock.tsx";
+
+export type StorageType = {
+    loadTeam: (setTeam: Dispatch<TeamType>) => () => void,
+    storeTeam: (team: TeamType | undefined) => () => void,
+    removeTeam: () => void,
+}
+
+const KEY = "team";
+export const LocalStorage = () => {
```

Comment:
Maybe we should rename it to `TeamStorage`?
I think is not relevant where you are persisting it (LocalStorage, SessionStorage, IndexedDB), but the fact that we are persisting Team is crucial because you are couple to the `KEY`